### PR TITLE
User edit count per namespace

### PR DESCRIPTION
**Pull request by alex-mashin**
Property type: record (formally page).
Property ID: '___USEREDITCNT'.
Property structure:
* '___USEREDITCNT_NS': _num (namespace number),
* '___USEREDITCNTNS_CNT': _num (user's edit count in this namespace).
Property domain: User namespace.
Property uniqueness: one value per namespace provided user has made edits in it.
Property annotator class: UserEditCountPropertyAnnotator.
Property annotator test class: UserEditCountPerNsPropertyAnnotatorTest.

{{#ask:}} query examples:

Edit count rating per user per namespace:
{{#ask:[[-User edit count per namespace::+]]
| mainlabel = -
| ?-User edit count per namespace
| ?Namespace
| ?Edit count
| sort = Edit count
| order = DESC
}}

Certain user's edits in the main namespace:
{{#ask:[[-User edit count per namespace::User:Alex Mashin]]
[[Namespace::0]]
| ?Edit count =
| mainlabel = -
}}

Made in reference to #169.

### CODE REVIEW

File: data/definitions.json [hunk 1]
```diff
@@ -141,6 +141,25 @@
...
+		"type": "_wpg",
+		"show": false,
+		"alias": "sesp-property-user-edit-count-per-ns",
+		"label": "User edit count per namespace"
+	},
+	"_USEREDITCNTNS_NS": {
+		"id": "___USEREDITCNTNS_NS",
+		"type": "_num",
+		"alias": "sesp-property-user-edit-count-ns",
+		"label": "User edit count per namespace:Namespace"
```
Comment (JeroenDeDauw) [inline data/definitions.json:155]: Why no space after the `:`? Like this the message confused me at first.

Comment (alex-mashin) [reply data/definitions.json:155]: As in `Exif` properties.

**Comment by alex-mashin** [comment]
Could anyone please approve the workflow and review the pull request?

**Comment by JeroenDeDauw** [comment]
(I do not currently have the capacity to review this)

**Comment by alex-mashin** [comment]
> (I do not currently have the capacity to review this)

Is there a way to invite amother reviewer then, e.g., @hexmode?

**Comment by alex-mashin** [comment]
Can anyone review this?

**Comment by krabina** [comment]
@gesinn-it-gea ?

**Comment by rotsee** [comment]
I haven't been coding for (S)MW in many, many years, so I'm not very useful as a reviewer, but I could add someone to the repo. @[JeroenDeDauw](https://github.com/JeroenDeDauw)?

**Pull request merged**